sdk/main/include/PublicKey.h

```diff
@@ -20,24 +20,19 @@
 #ifndef PUBLIC_KEY_H_
 #define PUBLIC_KEY_H_
 
-#include "Key.h"
+#include <proto/basic_types.pb.h>
 
 namespace Hedera
 {
-class PublicKey : public Key
+class PublicKey
 {
 public:
-  static PublicKey fromAliasBytes(const std::string& bytes)
-  {
-    (void)bytes;
-    return PublicKey();
-  }
+  PublicKey();
+  PublicKey(const PublicKey& other) = default;
+  PublicKey& operator=(const PublicKey& other) = default;
+  PublicKey& operator=(const PublicKey&& other) = delete;
 
-  bool operator==(const PublicKey& key) const
-  {
-    (void)key;
-    return true;
-  }
+  virtual proto::Key* toProtobuf() const { return new proto::Key(); }
```

Comment:
I know it's just stubbed but would you be able to put this definition in the source file? That way we can remove the inclusion of `<proto/basic_types.ph.b>` from this header file and keep the protobuf files unexposed.

Reply:
The stub definition is being deleted in my next PR - it becomes pure virtual, since all child classes of PublicKey must implement

```
virtual proto::Key* toProtobuf() const = 0;
```

That said, removing the definition in the header won't let us remove the include - the return type in proto will still need the include, no?

Reply:
You can forward declare the return type, so putting something like `namespace proto { class Key; }` at the top of the file (I've been putting my forward declarations right after the `#include`s) should be all that is needed.